gamestonk_terminal/cryptocurrency/due_diligence/glassnode_model.py

```diff
@@ -161,6 +161,49 @@
 INTERVALS = ["1h", "24h", "10m", "1w", "1month"]
 
 
+def get_non_zero_addresses(
+    asset: str, interval: str, since: int, until: int
+) -> pd.DataFrame:
+    """Returns addresses with non-zero balance of a certain asset
+    [Source: https://glassnode.com]
+
+    Parameters
+    ----------
+    asset : str
+        Asset to search (e.g., BTC)
+    since : int
+        Initial date timestamp (e.g., 1_577_836_800)
+    until : int
+        End date timestamp (e.g., 1_609_459_200)
+    interval : str
+        Interval frequency (e.g., 24h)
+
+    Returns
+    -------
+    pd.DataFrame
+        addresses with non-zero balances
+    """
+
+    url = api_url + "addresses/non_zero_count"
+
+    parameters = {
+        "api_key": cfg.API_GLASSNODE_KEY,
+        "a": asset,
+        "i": interval,
+        "s": str(since),
+        "u": str(until),
+    }
+
+    r = requests.get(url, params=parameters)
+    print(url, parameters)
```

Comment:
<img width="1208" alt="Screenshot 2021-12-30 at 02 24 39" src="https://user-images.githubusercontent.com/25267873/147716751-f9953b0d-fd97-4ae3-917d-7df4aaf98d6f.png">

random print in the wild!